File: routeMap/app/src/main/java/ru/android/zheka/db/DatabaseContentProvider.java
package ru.android.zheka.db;

import com.activeandroid.Configuration;
import com.activeandroid.content.ContentProvider;

import ru.android.zheka.db.Config;
import ru.android.zheka.db.Point;
import ru.android.zheka.db.Trace;

public class DatabaseContentProvider extends ContentProvider {

@Override
protected Configuration getConfiguration() {
    Configuration.Builder builder = new Configuration.Builder(getContext());
    builder.addModelClass(Config.class);
    builder.addModelClass(Trace.class);
    builder.addModelClass(Point.class);
    return builder.create();
 }
}

File: routeMap/app/src/main/java/ru/android/zheka/db/DatabaseContentProvider.kt
package ru.android.zheka.db

import com.activeandroid.Configuration
import com.activeandroid.content.ContentProvider

class DatabaseContentProvider : ContentProvider() {
    override fun getConfiguration(): Configuration {
        val builder = Configuration.Builder(context)
        builder.addModelClass(Config::class.java)
        builder.addModelClass(Trace::class.java)
        builder.addModelClass(Point::class.java)
        return builder.create()
    }
}